### ISSUE TITLE
Allow native events to be propagated to consumer of components

### ISSUE THREAD

**Issue by Amitesh** [ng-select/ng-select#827]
Hi,

**Can we remove `event.stopPropagation()` from events which open the door of posibilities**
I am working on a project, where we need to implement accessibility and when user hit the enter on remove (cross X) icon then it will clear the content. I am ware that it is happening on `delete` key press but my accessibility test is getting failed.

Is it possible to remove the `stopPropagation()` call from `_handleEnter()` function. I will capture the event by adding event listener on the element. If I get an api for that event then it will be nothing like that.

Thanks for wonderful library!

Kind Regards,
Amitesh

**Comment by anjmao** [comment]
Hi @Amitesh,

If I remember correctly it can break other functionality, it would be great if you can remove stopPropagation locally and test if it doesn't break something :)

**Comment by varnastadeus** [comment]
fixed in https://github.com/ng-select/ng-select/pull/1174
